models/stg_snapchat.yml

```diff
@@ -279,27 +291,27 @@ models:
       - name: quartile_3
         description: The number of times your video ad was viewed to 75% 
       - name: saves
-        description: Number of times a lens/filter was saved to Memories
+        description: Number of times components of the campaign were saved to Memories
       - name: shares
-        description: Number of times a lens/filter was shared in a Chat or Story
+        description: Number of times components of the campaign were shared in a Chat or Story
       - name: screen_time
-        description: The number of milli seconds spent watching your ad across all paid impressions. Screen time starts recording as soon as the media is fully rendered on the device and the autoplay video starts
+        description: The number of seconds spent watching your ad across all paid impressions. Screen time starts recording as soon as the media is fully rendered on the device and the autoplay video starts
       - name: video_views
         description: The number of times your video was watched at least 2 seconds or swiped up, whichever comes first
       - name: view_completion
         description: The number of times your video ad was viewed to completion (97% )
       - name: view_time
-        description: The number of milli seconds spent watching your ad across all users reached.
+        description: The number of seconds spent watching your ad across all users reached.
       - name: impressions
         description: The number of impressions for an ad in the hour of the record.
       - name: spend
         description: The amount of spend for an ad in the hour of the record.
       - name: swipes
         description: The number of swipes for an ad in the hour of the record. Swipes are when your ad was swiped up on or the CTA was tapped to view the attachment below.
+      - name: conversion_purchases
+        description: The total number of purchases attributed to an ad for the day of the record.
       - name: conversion_purchases_value
         description: The total value of purchases attributed to an ad in the hour of the record (in Ad Account's currency, converted from microcurrency).
-      - name: conversion_purchases
```

Comment:
Same comment: Can you add `conversion_purchases` back in for all the relevant stg models here? It is included by default as part of the [conversion_fields](https://github.com/fivetran/dbt_snapchat_ads_source/tree/main?tab=readme-ov-file#configuring-conversion-fields)

Reply:
i believe this was done but lmk if you don't see the changes